### PR TITLE
Add additional log levels and tests

### LINKED ISSUE #1: Additional log levels + info
Would be helpful to do info, warning, etc messages as well as pass additional data and a message along with an exception

### PR DESCRIPTION
**Pull request by nickstanish**
Closes #1

### CODE REVIEW

File: lib/rollbar_helper/version.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module RollbarHelper
+  VERSION = '0.2.1'
```
Comment (santi-h) [inline lib/rollbar_helper/version.rb]: I think this should up the minor instead though

Comment (nickstanish) [reply lib/rollbar_helper/version.rb]: You’re right, I wasn’t sure if you said minor or patch this morning but I thought minor made more sense if this is following semver

Comment (nickstanish) [reply lib/rollbar_helper/version.rb]: Fixed

Comment (santi-h) [reply lib/rollbar_helper/version.rb]: 👍

**Pull request merged**